src/js/utils/browser.js

```diff
@@ -48,10 +51,10 @@ export const ANDROID_VERSION = (function() {
     return parseFloat(match[1] + '.' + match[2]);
   } else if (major) {
     return major;
-  } else {
-    return null;
   }
-})();
+  return null;
+}());
```

Comment:
It came all the way from the original fork of standard and _you_ specifically changed it to `"outside"`. 😉